src/hotspot/cpu/x86/x86.ad

```diff
@@ -1227,10 +1227,12 @@ static inline Assembler::AvxVectorLen vector_length_encoding(const MachNode* use
 }
 
 static inline bool is_unsigned_booltest_pred(int bt) {
-  return  ((bt & BoolTest::unsigned_compare) == BoolTest::unsigned_compare);
+  return  (bt == BoolTest::ule ||
```

Comment:
You could just test if the unsigned_compare bit is set?

Reply:
Done